src/components/App/App.js

```diff
@@ -51,6 +52,8 @@ function App() {
 
   const startScanning = () => {
     setIsScanning(true);
+    setQrData(null);
+
```

Comment:
The comment is for line 67 - but I can't put a comment on it. For the video.play, we probably need a catch block. Currently obtaining two errors: 
![Screen Shot 2021-08-18 at 9 39 42 AM](https://user-images.githubusercontent.com/65190574/129918488-cdf697a7-3a7a-49c5-ab9c-c0490aaff621.png)

Reply:
I can add something to catch the exception, when do you get this error?

Reply:
If you start the scan and stop it - it should pop up sometimes happens on the second attempt

Reply:
Should be resolved now.